frontend/src/app/search/page.tsx

```diff
@@ -22,40 +18,46 @@ const searchFetcher: SearchFetcher = useMockData
 //   locale: string;
 // }
 
-export default function Search() {
-  const { featureFlagsManager, mounted } = useFeatureFlags();
-  const [searchResults, setSearchResults] = useState<SearchResponseData>([]);
-
-  const handleButtonClick = (event: React.MouseEvent<HTMLButtonElement>) => {
-    event.preventDefault();
-    performSearch().catch((e) => console.log(e));
-  };
-
-  const performSearch = async () => {
-    const opportunities = await fetchSearchOpportunities(searchFetcher);
-    setSearchResults(opportunities);
-  };
+async function SearchResults() {
+  const results = await fetchSearchOpportunities(searchFetcher);
+  return (
+    <ul>
+      {results.map((opportunity) => (
+        <li key={opportunity.opportunity_id}>
+          {opportunity.category}, {opportunity.opportunity_title}
+        </li>
+      ))}
+    </ul>
+  );
+}
 
-  if (!mounted) return null;
-  if (!featureFlagsManager.isFeatureEnabled("showSearchV0")) {
-    return <PageNotFound />;
+export default function Search() {
+  async function updateResults(formData: FormData) {
+    // server action
+    "use server";
+    console.log(Object.fromEntries(formData.entries()));
+    await new Promise((resolve) => setTimeout(resolve, 750));
+    revalidatePath("/search");
```

Comment:
Believe the convention is to move server actions to  `/app/searchactions.ts` - just keeping everything in one file to keep it simple for now.